lib/cable_ready/engine.rb

```diff
@@ -49,6 +49,20 @@ class Engine < Rails::Engine
       end
     end
 
+    initializer "cable_ready.integration_test_request_encoding" do
+      ActiveSupport.on_load(:action_dispatch_integration_test) do
+        # Support `as: :cable_ready`. Public `register_encoder` API is a little too strict.
+        class ActionDispatch::RequestEncoder
+          class CableReadyStreamEncoder < IdentityEncoder
+            header = [ Mime[:cable_ready], Mime[:html] ].join(",")
```

Comment:
Hey @tcannonfodder, thanks for this pull request. Should this be `:json` instead of `:html`, since the MIME-Type is `text/vnd.cable-ready.json`:

```suggestion
            header = [ Mime[:cable_ready], Mime[:json ].join(",")
```

Reply:
Yes, you are totatally @kaspth! Thanks!